app/mailers/report_mailer.rb

```diff
@@ -1,30 +1,32 @@
 class ReportMailer < ActionMailer::Base
-  default :from => AppConfig.mail.sender_address
+  default from: AppConfig.mail.sender_address
 
-  def new_report(type, id)
+  def self.new_report(type, id)
```

Comment:
Personally I'd prefer the use of "module_function" instead of the "self" syntax but maybe that's just me.:)